src/server.js

```diff
@@ -84,13 +93,17 @@ module.exports = function() {
 	manager.sockets = sockets;
 
 	const protocol = config.https.enable ? "https" : "http";
-	const host = config.host || "*";
 
 	log.info(`The Lounge ${colors.green(Helper.getVersion())} is now running \
 using node ${colors.green(process.versions.node)} on ${colors.green(process.platform)} (${process.arch})`);
 	log.info(`Configuration file: ${colors.green(Helper.CONFIG_PATH)}`);
-	log.info(`Available on ${colors.green(protocol + "://" + host + ":" + config.port + "/")} \
-in ${config.public ? "public" : "private"} mode`);
+	if (typeof config.host !== "undefined" && config.host.includes("unix:")) {
+		log.info(`Available on socket ${colors.green(config.host)} \
```

Comment:
I did the other two changes but this one doesn't seem to be that simple as if I just change the protocol to unix or unixs, the protocol will be shown twice as it is already included in the config.host value. We'll either need to do some if-elses and/or search-replaces or just stick with two messages.